src/components/tiles/CryptoFiatAmountTile.js

```diff
@@ -1,28 +1,31 @@
 // @flow
+/* eslint-disable react-native/no-raw-text */
```

Comment:
Do you remember the edit I made to `.eslintrc.json`? If we do those, we don't need these `eslint-disable` lines.

Reply:
Ok, we can try some things and submit it as another PR.